src/collections/_documentation/performance-monitoring/performance/index.md

```diff
@@ -38,4 +38,4 @@ Below the graphs are a list of transactions that may be filtered down based on g
 
 ## Key Transaction View
 
-If you have transactions you frequently return to;  you can mark each one off as a key transaction in their corresponding [summary views](/performance-monitoring/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+If you have transactions you frequently return to;  you can mark each one as a key transaction in their corresponding [summary views](/performance-monitoring/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
```

Comment:
Suggest: "If you have transactions to which you frequently return, you can mark each as a key transaction in..."